### ISSUE TITLE
bad replica declaration failing in kronos

### ISSUE THREAD

**Issue by tbeerman** [rucio/rucio#1766]
Motivation
----------
Bad replicas declaration in kronos currently fails with an exception which slows down kronos a lot:

```
Exception in thread StompReceiverThread-114666:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 812, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 765, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib/python2.7/site-packages/stomp/transport.py", line 342, in __receiver_loop
    self.process_frame(f, frame)
  File "/usr/lib/python2.7/site-packages/stomp/transport.py", line 191, in process_frame
    self.notify(frame_type, f.headers, f.body)
  File "/usr/lib/python2.7/site-packages/stomp/transport.py", line 245, in notify
    rtn = notify_func(headers, body)
  File "/usr/lib/python2.7/site-packages/rucio/daemons/tracer/kronos.py", line 126, in on_message
    self.__update_atime()
  File "/usr/lib/python2.7/site-packages/rucio/daemons/tracer/kronos.py", line 150, in __update_atime
    logging.error('Failed to declare suspicious file' + error)
TypeError: cannot concatenate 'str' and 'exceptions.TypeError' objects
```

Modification
------------
Convert correctly to str

**Comment by cserf** [comment]
Hmmm... This one is a blocker for Kronos. I did a manual patch on the nodes, and I expected it would be in 1.18.5

**Comment by cserf** [comment]
Actually this has been fixed by #1770
